sormas-backend/src/test/java/de/symeda/sormas/backend/visit/ExternalVisitTest.java

```diff
@@ -0,0 +1,537 @@
+		person.setBirthdateDD(1);
+		person.setSymptomJournalStatus(SymptomJournalStatus.REGISTERED);
+		final ContactDto contact1 =
+			creator.createContact(externalVisitsUser.toReference(), person.toReference(), DateHelper.subtractDays(new Date(), 41));
+		final ContactDto contact2 =
+			creator.createContact(externalVisitsUser.toReference(), person.toReference(), DateHelper.subtractDays(new Date(), 29));
+		contact1.setOverwriteFollowUpUntil(true);
+		contact2.setOverwriteFollowUpUntil(true);
+
+		Date now = new Date();
+		contact1.setFollowUpUntil(DateHelper.subtractDays(now, 20));
+		contact2.setFollowUpUntil(DateHelper.subtractDays(now, 8));
+
+		getPersonFacade().savePerson(person);
+		getContactFacade().save(contact1);
+		getContactFacade().save(contact2);
+
+		loginWith(externalVisitsUser);
+
+		JournalPersonDto exportPerson = getPersonFacade().getPersonForJournal(person.getUuid());
+		assertEquals(person.getFirstName(), exportPerson.getFirstName());
+		assertEquals(person.getLastName(), exportPerson.getLastName());
+		assertEquals(person.getSex(), exportPerson.getSex());
+		assertEquals(person.getEmailAddress(), exportPerson.getEmailAddress());
+		assertEquals(internationalPhoneNumber, exportPerson.getPhone());
+		assertEquals(person.getBirthdateYYYY(), exportPerson.getBirthdateYYYY());
+		assertEquals(person.getBirthdateMM(), exportPerson.getBirthdateMM());
+		assertEquals(person.getBirthdateDD(), exportPerson.getBirthdateDD());
+		assertEquals(contact2.getFollowUpUntil(), exportPerson.getLatestFollowUpEndDate());
+	}
+
+	@Test
+	/*
+	 * If you need to change this test to make it pass, you probably changed the behaviour of the ExternalVisitsResource.
+	 * Please note that other system used alongside with SORMAS are depending on this, so that their developers must be notified of any
+	 * relevant API changes some time before they go into any test and productive system. Please inform the SORMAS core development team at
+	 * https://gitter.im/SORMAS-Project!
+	 */
+	public void testIsValidPersonUuid() {
+		final PersonDto person = creator.createPerson("James", "Smith", Sex.MALE, 1980, 1, 1);
+
+		loginWith(externalVisitsUser);
+
+		assertTrue(getPersonFacade().isValidPersonUuid(person.getUuid()));
+		assertFalse(getPersonFacade().isValidPersonUuid("2341235234534"));
+	}
+
+	@Test
+	/*
+	 * If you need to change this test to make it pass, you probably changed the behaviour of the ExternalVisitsResource.
+	 * Please note that other system used alongside with SORMAS are depending on this, so that their developers must be notified of any
+	 * relevant API changes some time before they go into any test and productive system. Please inform the SORMAS core development team at
+	 * https://gitter.im/SORMAS-Project!
+	 */
+	public void testSetSymptomJournalStatus() {
+		PersonDto person = creator.createPerson();
+
+		loginWith(externalVisitsUser);
+		getPersonFacade().setSymptomJournalStatus(person.getUuid(), SymptomJournalStatus.REGISTERED);
```

Comment:
This method calls the [PersonFacadeEjb.savePerson(...)](https://github.com/hzi-braunschweig/SORMAS-Project/blob/7a4043b2df3617f24119e4ee541e8c2bf980ac06/sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java#L418), which is annotated with `@RightsAllowed(UserRight._PERSON_EDIT)`. 
Isn't this a problem? The External_Visits_User does not have  this right by default. I wonder why this test passes...